Use common options for `cp` command to be compatible with the Busybox cp.

--- a/Makefile.am
+++ b/Makefile.am
@@ -1,9 +1,9 @@
 EXTRA_DIST=autogen.sh AUTHORS COPYING
 
 dist-hook:
-	cp -raf matefaenza $(distdir)/matefaenza
-	cp -raf matefaenzadark $(distdir)/matefaenzadark
-	cp -raf matefaenzagray $(distdir)/matefaenzagray
+	cp -rf matefaenza $(distdir)/matefaenza
+	cp -rf matefaenzadark $(distdir)/matefaenzadark
+	cp -rf matefaenzagray $(distdir)/matefaenzagray
 
 	@if test -d "$(srcdir)/.git"; \
 	then \
@@ -25,6 +25,6 @@
 
 install-data-hook:
 	$(mkdir_p) $(DESTDIR)$(datadir)/icons
-	cp -raf --no-preserve=ownership matefaenza $(DESTDIR)$(datadir)/icons
-	cp -raf --no-preserve=ownership matefaenzadark $(DESTDIR)$(datadir)/icons
-	cp -raf --no-preserve=ownership matefaenzagray $(DESTDIR)$(datadir)/icons
+	cp -rf matefaenza $(DESTDIR)$(datadir)/icons
+	cp -rf matefaenzadark $(DESTDIR)$(datadir)/icons
+	cp -rf matefaenzagray $(DESTDIR)$(datadir)/icons
